browser-test/src/applicant_application_index.test.ts

```diff
@@ -90,4 +94,56 @@ describe('applicant program index page', () => {
       wantSubmittedPrograms: [],
     })
   })
+
+  it('shows previously answered on text for program that was previously applied to', async () => {
+    const {page, applicantQuestions} = ctx
+
+    await loginAsTestUser(page)
+    await selectApplicantLanguage(
+      page,
+      'English',
+      /* assertProgramIndexPage= */ true,
+    )
+
+    // Fill out application with one question and confirm it was successfully submitted.
+    await applicantQuestions.applyProgram(otherProgramName)
+    await applicantQuestions.answerTextQuestion('first answer')
+    await applicantQuestions.clickNext()
+    await applicantQuestions.submitFromReviewPage()
+    await applicantQuestions.returnToProgramsFromSubmissionPage()
+    await applicantQuestions.expectPrograms({
+      wantNotStartedPrograms: [primaryProgramName],
+      wantInProgressPrograms: [],
+      wantSubmittedPrograms: [otherProgramName],
+    })
+
+    // Check that the question repeated in the program with two questions shows previously answered.
+    await applicantQuestions.clickApplyProgramButton(primaryProgramName)
+    await validateScreenshot(page, 'question-shows-previously-answered')
+
+    // Fill out second question.
+    await applicantQuestions.clickContinue()
+    await applicantQuestions.answerTextQuestion('second answer')
+    await applicantQuestions.clickNext()
+    await applicantQuestions.submitFromReviewPage()
+
+    // Check that the original program doesn't show previously answered.
+    await applicantQuestions.returnToProgramsFromSubmissionPage()
+    await applicantQuestions.clickApplyProgramButton(otherProgramName)
+    await validateScreenshot(page, 'question-does-not-show-previously-answered')
+    await applicantQuestions.clickSubmit()
+
+    // Change first response on second program.
+    await applicantQuestions.returnToProgramsFromSubmissionPage()
+    await applicantQuestions.clickApplyProgramButton(primaryProgramName)
+    await applicantQuestions.clickEdit()
+    await applicantQuestions.answerTextQuestion('first answer 2')
+    await applicantQuestions.clickNext()
+    await applicantQuestions.submitFromReviewPage()
+
+    // Check that the first program now shows the previously answered text.
+    await applicantQuestions.returnToProgramsFromSubmissionPage()
+    await applicantQuestions.clickApplyProgramButton(otherProgramName)
+    await validateScreenshot(page, 'first-program-shows-previously-answered')
```

Comment:
**issue (blocking):** "first" won't make sense among all the screenshots, and isn't readable for what it's indicating, plus as far as a screenshot it's not important to its use, you could likely use it in different situations where it isn't the first.  It is a little tricky since there is some context in the test that usually matters but not always for the suite.

Suggestion: "other-program-shows-previously-answered-on"

Reply:
Yeah, makes sense! Updated.